acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/models/TimeSeriesBuilder.java

```diff
@@ -0,0 +1,80 @@
+package uk.nhs.digital.ps.test.acceptance.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toList;
+
+@SuppressWarnings("WeakerAccess") // builder's methods are intentionally public
+public class TimeSeriesBuilder {
+
+    private String title;
+    private List<PublicationBuilder> publicationBuilders = new ArrayList<>();
+    private String name;
+
+    public static TimeSeriesBuilder newTimeSeries() {
+        return new TimeSeriesBuilder();
+    }
+
+    //<editor-fold desc="BUILDER METHODS">
+    public TimeSeriesBuilder withName(final String name) {
+        return cloneAndAmend(builder -> builder.name = name);
```

Comment:
I like this :-)

Reply:
I liked your idea of immutable builders :-)